src/components/Contentful/StoryCard.vue

```diff
@@ -8,14 +8,21 @@
 			:class="themeClass"
 			v-if="hasBackgroundImage"
 		>
-			<kv-contentful-img
-				class="kv-contentful-img tw-h-full tw-w-full tw-object-cover"
-				:width="520"
-				fallback-format="jpg"
-				:contentful-src="backgroundImage.url"
-				:alt="backgroundImage.description"
-			/>
-			<p class="story-card__imageCard-title tw-text-h4">
+			<component
+				:is="cardLink ? 'a' : 'div'"
+				:href="cardLink ? cardLink : null"
+				v-kv-track-event="cardAnalytics"
+				class="tw-block"
+			>
+				<kv-contentful-img
+					class="kv-contentful-img tw-h-full tw-w-full tw-object-cover"
+					:width="520"
+					fallback-format="jpg"
+					:contentful-src="backgroundImage.url"
+					:alt="backgroundImage.description"
+				/>
+			</component>
+			<p class="story-card__imageCard-title tw-text-h4" v-if="!cardLink">
```

Comment:
Why would we hide this due to having a link on the card? Seems like we'd still like to show this interface element if it's present...not a blocker but likely a point of confusion for future usage.

Reply:
I can double check with stakeholders.